GoMeet/src/main/webapp/js/email-dao.js

```diff
@@ -0,0 +1,35 @@
+class EmailDAO {
+  /** Data Access Object for sending emails */
+
+  static endpoint = '/email'; // URL endpoint of the associated servlet
+  
+  /**
+   * Completes a POST request to send an email invitation with the 
+   * meeting event link to all addresses in the guest list 
+   * @param {String} meetingTimeId the unique entity key of the 
+   * MeetingEvent that the guest list is invited to 
+   * @param {List} guestList the list of guests' email addresses 
+   * @return JSON containing the sent status of the emails  
+   */
+  static async inviteGuestsToMeeting(meetingEventId, guestList) {
+    if (meetingEventId === null || meetingEventId === undefined) {
+      throw new Error(INSUFFICIENT_REQUEST_PARAM);
+    }
+
+    if (guestList === null || guestList === undefined) {
```

Comment:
This if-block and the above if-block can be combined into one for brevity. This is what Daniel mentioned in my previous [PR](https://github.com/googleinterns/step271-2020/pull/32) 👍